src/selector.js

```diff
@@ -0,0 +1,35 @@
+const parser = require('postcss-selector-parser');
+
+const caseSensitivePseudoClassesLists = [
+  ':active',
+  ':hover',
+  ':focus',
+  ':focus-within',
+  ':visited',
+];
+
+const caseSensitivePseudoClassesSet = new Set(caseSensitivePseudoClassesLists);
+
+function getParsed(selectors, callback) {
+  return parser(callback).processSync(selectors);
+}
+
+export const transformer = (str) =>
+  getParsed(str, (sel) => {
+    sel.walkTags((selNode) => {
+      selNode.value = selNode.value.toLowerCase();
+
+      return selNode;
+    });
+    sel.walkAttributes((selNode) => {
+      selNode._attribute = selNode._attribute.toLowerCase();
+    });
+    sel.walkPseudos((selNode) => {
+      const normalizeVal = `${selNode.value.toLowerCase()}`;
+      if (!caseSensitivePseudoClassesSet.has(normalizeVal)) {
```

Comment:
Need lowercase `::before`/`::after` and other known pseudo element

Reply:
They are case insensitive so if someone writes them in CAPITAL, then it's wrong CSS

Reply:
what is pseudo-elements is case sensitivity?

Reply:
Active, focus, focus-within, visited, hover doesn't work when they are in caps

Reply:
Rest all are case insensitive

Reply:
Try:
```css
      div::BEFORE {
        content: "tets\6D";
      }
```

works fine

Reply:
yeah but

```css
a:hover {
  color: red;
}
```

Wont work thats why I kept them in a list which are case sensitive

https://github.com/anikethsaha/postcss-lowercase-text/pull/3/files#diff-f67fc5ea4e003441c384bdf18e81ba52R3-R9

```css
div::before {
  content: "tets\6D";
}
```

Will get transformed to

```css
div::before {
  content: "tets\6D";
}
```